CONTRIBUTING.md

```diff
@@ -57,8 +57,19 @@ You can add the sign off when creating the git commit via `git commit -s`.
 
 ## Code Reviews
 
-* A single approval is sufficient to merge. If a reviewer asks for changes in a PR they should be
-addressed before the PR is merged, even if another reviewer has already approved the PR.
-* During the review, address the comments and commit the changes _without_ squashing the commits.
-This facilitates incremental reviews since the reviewer does not go through all the code again to
-find out what has changed since the last review.
+* The pull request title should describe what the change does and not embed issue numbers.
+The pull request should only be blank when the change is minor. Any feature should include
+a description of the change and what motivated it. If the change or design changes through
+review, please keep the title and description updated accordingly.
+* A single approval is sufficient to merge. If a reviewer asks for
+changes in a PR they should be addressed before the PR is merged,
+even if another reviewer has already approved the PR.
+* During the review, address the comments and commit the changes
```

Comment:
I think I would be a lot more happy with the "no squash" policy if we say, "once you have a review" please don't squash

because it is annoying especially in the case where no one is around and they haven't reviewed yet either. The whole point it to prevent re-review.

So, how about we clarify as that matches the intent?

Reply:
this would implicitly cover "draft" PRs as typically folks don't review them in draft state.

Reply:
or maybe 'During the review" says that anyway, if we think it does nevermind!

Reply:
I think `During the review` says that :D